src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -0,0 +1,48 @@
+package com.kodilla.ecommercee.domain;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity(name = "users")
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "username")
+    private String username;
+
+    @Column(name = "status")
+    private String status;
+
+    @Column(name = "userKey")
+    private Long userKey;
+
+    @Column(name = "address")
+    private String address;
+
+    @Column(name = "email")
+    private String email;
+
+    @Column(name = "phoneNumber")
+    private String phoneNumber;
+
+    @OneToMany(
+            targetEntity = User.class,
```

Comment:
Czy tutaj na pewno będzie User.class? Bardziej bym to widział, że jeden User może mieć wiele Order. :-) Brak cascade type oraz mappedBy. Ale to możemy ustalić wspólnie, jeżeli także uważasz, że połączenie będzie z Order. :-)

Reply:
Generalnie w tym momencie to wygląda tak jakbyś chciał pobierać listę userów do klasy User? Oo

Reply:
Zmieniłem na Order.class. Dodałem cascade type oraz mappedBy. Czy coś jeszcze Waszym zdaniem powinienem tutaj poprawić ?

Reply:
Na razie wygląda ok. Przy testach zobaczymy, jak pójdzie. :-)